app/src/main/java/com/sun/noteapp/ui/todonote/ToDoNoteActivity.kt

```diff
@@ -0,0 +1,252 @@
+                buttonAlarmToDoNote.text = it.remindTime
+            }
+            editTitleTextNote.apply {
+                setText(it.title)
+                setSelection(it.title.length)
+            }
+            if (it.remindTime != NONE) buttonAlarmToDoNote.text = it.remindTime
+            textUpdateTime.text = it.modifyTime
+            color = it.color
+            password = it.password
+            if (it.label != NONE) listLabel =
+                ConvertString.labelStringDataToLabelList(it.label)
+            labelAdapter.submitList(listLabel)
+            toDos.addAll(ConvertString.actionDataStringToActionCheckList(it.content))
+        }
+        updateView(color)
+    }
+
+    private fun setUpEventMoveToDo() {
+        val moveCallBack = object : ItemTouchHelper.Callback() {
+            override fun getMovementFlags(
+                recyclerView: RecyclerView,
+                viewHolder: RecyclerView.ViewHolder
+            ): Int =
+                makeMovementFlags(ItemTouchHelper.UP or ItemTouchHelper.DOWN, 0)
+
+            override fun onMove(
+                recyclerView: RecyclerView,
+                viewHolder: RecyclerView.ViewHolder,
+                target: RecyclerView.ViewHolder
+            ): Boolean {
+                return true
+            }
+
+            override fun onSwiped(viewHolder: RecyclerView.ViewHolder, direction: Int) {
+            }
+        }
+        itemTouchHelper = ItemTouchHelper(moveCallBack)
+        itemTouchHelper?.attachToRecyclerView(recyclerToDo)
+    }
+
+    private fun openDialog(itemId: Int?) = when (itemId) {
+        R.id.bottomMenuDelete -> {
+            if (id != 0) showDeleteDialog()
+            true
+        }
+        R.id.bottomMenuLock -> showPassDialog()
+        R.id.bottomMenuDeleteRemind -> {
+            if (buttonAlarmToDoNote.text.isNotEmpty()) showDeleteAlarmDialog()
+            true
+        }
+        else -> false
+    }
+
+    private fun showDeleteAlarmDialog() {
+        AlertDialog.Builder(this)
+            .setMessage(R.string.message_alarm_off)
+            .setPositiveButton(R.string.button_yes) { _, _ ->
+                buttonAlarmToDoNote.text = null
+                remindTime = NONE
+            }
+            .setNegativeButton(R.string.button_cancel) { _, _ -> }
+            .show()
+    }
+
+    private fun showPassDialog(): Boolean {
+        val input = EditText(this)
+        input.apply {
+            layoutParams = ConstraintLayout.LayoutParams(
+                ConstraintLayout.LayoutParams.MATCH_CONSTRAINT,
+                ConstraintLayout.LayoutParams.MATCH_CONSTRAINT
+            )
+            inputType = InputType.TYPE_TEXT_VARIATION_PASSWORD
+            transformationMethod = PasswordTransformationMethod.getInstance()
+            if (password != NONE) {
+                setText(password)
+                setSelection(password.length)
+            }
+        }
```

Comment:
tách ra hàm riêng để tạo cái này, mỗi hàm chỉ làm 1 việc thôi
```suggestion
       val input = EditText(this).apply {
```